src/main/java/entities/Player.java

```diff
@@ -0,0 +1,5 @@
+package entities;
+
+public class Player extends User{
+
```

Comment:
add the following attributes:
- prevPlayedMazes: a string array of ids of played mazes
- addPlayedMaze: a function to add a new maze id to the prevPlayedMaze array